docs/en/operations/system-tables.md

```diff
@@ -5,11 +5,40 @@ toc_title: System Tables
 
 # System Tables {#system-tables}
 
-System tables are used for implementing part of the system’s functionality, and for providing access to information about how the system is working.
-You can’t delete a system table (but you can perform DETACH).
-System tables don’t have files with data on the disk or files with metadata. The server creates all the system tables when it starts.
-System tables are read-only.
-They are located in the ‘system’ database.
+## Introduction
+
+System tables provide:
+
+- Information about server states, processes and environment.
+- Support for server's internal processes.
+
+System tables:
+
+- Don't store data and metadata in the storage filesystem. ClickHouse server creates system tables at start.
```

Comment:
Afaik not always true, some tables are persisted.

Reply:
query_log, metric_log, trace_log and probably others store data on disk. You can check data directory if you have ClickHouse running